### PR TITLE
Update the docs

### PR DESCRIPTION
**Pull request by daggintosh**
In progress of reformatting the docs and writing the build instructions.

### CODE REVIEW

**Review by sudokoko — COMMENTED** [review]
README.md looks good, let me know if you want assistance with BUILDING.md

File: BUILDING.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+## Hardware Requirements:
+- 64-bit x86 or ARM CPU
+- 1 GB RAM 
+
+MySQL is very intensive, you should be able to run this on even a potato though!
+## Software Requirements:
+- .NET 7 SDK
+- ASP.NET 7 Runtime
+- MySQL or MariaDB
+- A will to play LBP after 2021
```
Comment (sudokoko) [inline BUILDING.md:14]: :sob:

File: README.md [hunk 2]
```diff
@@ -18,18 +18,16 @@ on our forum.
 ## DISCLAIMERS (Please read!)
 
 ### This is not a final product.
-This is **beta software**, and thus is **not stable nor is it secure**.
+This is **beta software**, and thus is potentially **unstable or insecure**.
 
 While Project Lighthouse is in a mostly working state, **we ask that our software not be used in a production
-environment until release**.
+environment without prioritizing security.**
```
Comment (Zaprit) [inline README.md:24]: maybe provide some example of what this means (external rate limiting, ddos protection, running on a box you don't care about on a network you don't care about, not storing your banking information in an lbp level)

File: BUILDING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# emerge --ask --verbose dotnet-sdk-bin mysql
+```
+## Building
+Clone `Project Lighthouse`
+```
+$ git clone https://github.com/LBPUnion/ProjectLighthouse.git
+```
+Build for `Production`
+```
+$ dotnet build -c Release
```
Comment (Zaprit) [inline BUILDING.md:37]: does this also restore packages, i remember that at least on mac getting dotnet to properly restore packages is a pain

**Review by Zaprit — COMMENTED** [review]
Maybe some changes on the wording of the two documents, I think this will minimise people asking questions like "what do i need to do to make lighthouse more safe".

**Comment by daggintosh** [comment]
closing this for now

**Pull request closed without merging**